server/certstore/certstore_windows.go

```diff
@@ -186,6 +218,14 @@ func TLSConfig(certStore StoreType, certMatchBy MatchByType, certMatch string, c
 		if pk == nil {
 			return ErrNoPrivateKeyStoreRef
 		}
+		// Look for CA Certificates
+		if caCertsMatch != nil {
```

Comment:
Would be safer to do `if len(caCertsMatch) != 0` which works on `nil` slices.

Reply:
Will fix, thanks Byron.